included the faq for gcc1.0 and ship-hats users and updated account f… (#46)
GovTechSG/techpass-user-docs: docs/support/user-account-migration.md

> # Common issues for GCC 1.0 and SHIP-HATS users

Reviewer: ```suggestion
# Common issues for GCC 1.0 MDM onboarding and SHIP-HATS account migration users
```

1. I received an email stating, "Your account was used for a new registration request". What does it mean?

You may receive this email if you are a GCC 1.0 user or part of the SHIP-HATS account migration, who tried to sign up for TechPass account when you already have an active account.

> If you did not re-attempt to sign up for TechPass account, this could be a potential hack and hence [report this incident][service-request] to us and monitor your TechPass account for suspicious activities.

Reviewer: ```suggestion
If you did not re-attempt to sign up for TechPass account nor part of the MDM onboarding or account migration exercise; Please [report this incident][service-request] to us and monitor your TechPass account for suspicious activities.
```

Author: done

When you request for a TechPass account, it is created and an email with an onboarding link is sent to you to complete the TechPass onboarding process. If you had not completed this within 30days using this link, you get this email.

To get a new invitation link, raise a [service request][service-request] with us. Once you receive a new invitation link, make sure to onboard in to TechPass within 30 days.

4. I am a public officer. How do I reset my TechPass account password?

If you are a public officer, generally, your organisational email address is your TechPass account. For example, *your_name<span>@</span>agency.gov.sg* or *your_name<span>@</span>tech.gov.sg*. Hence, your GSIB device password is also your TechPass account password. If you are currently logged in to your GSIB device, proceed to [reset your password][reset-password-gsib]. If there are any issues, contact your Agency Facility Management (AFM).

5. I am a vendor. How do I reset password for my TechPass account?

Generally, if you are a vendor, your TechPass ID will look like *your_name<span>@</span>techpass.gov.sg*.
[Reset your password][reset-password-vendor] by following the onscreen instructions on this page.

> 6. why do public officers have 2 MFA profiles, SG Govt M365 and TECHPASS, on their authenticator app?

Reviewer: ```suggestion
6. Why do public officers have 2 MFA profiles, SG Govt M365 and TECHPASS, on their authenticator app?
```

> If you are a public officer, generally, your organisational email address is your TechPass account. For example, *your_name<span>@</span>agency.gov.sg* or *your_name<span>@</span>tech.gov.sg*.

Reviewer: ```suggestion
If you are a public officer; your organisational email address is your TechPass account. For example, *your_name<span>@</span>agency.gov.sg* or *your_name<span>@</span>tech.gov.sg*.
```

Author: done.

> While SG Govt M365 profile is linked to your WOG email account, TECHPASS profile is linked to your TechPass account which in turn is associated with your organisational email address.

Author: Since the users can also be vendors - agency email address - is it applicable?
 SG Govt M365 MFA profile is linked to your official email address and TECHPASS MFA profile is linked to your TechPass account which is also associated with your official email address.

> While SG Govt M365 profile is linked to your WOG email account, TECHPASS profile is linked to your TechPass account which in turn is associated with your organisational email address.

Reviewer: right.. instead of specifying agency/organisation emails.. its definitely easier to just refer to official email

> While SG Govt M365 profile is linked to your WOG email account, TECHPASS profile is linked to your TechPass account which in turn is associated with your organisational email address.

Author: done, thanks!

> If you are using TechPass as an SSO to access any SGTS services from your non-GSIB device, first you need to authenticate your WOG account by entering the OTP displayed for your SG Govt M365 profile on the Authenticator app. Once this is successful, depending on the MFA settings you configured earlier, you will be prompted to approve your TechPass account.

Reviewer: ```suggestion
If you are using TechPass as an SSO to access any SGTS services from your GMD device; First, you need to authenticate your WOG account by entering your email and the OTP as verification code displayed in your SG Govt M365 profile on the Authenticator app. Once this is successful, depending on the MFA settings you configured earlier, you will be prompted to approve your TechPass account.
```

# Common issues for GCC 1.0 MDM onboarding and SHIP-HATS account migration users

1. I received an email stating, "Your account was used for a new registration request". What does it mean?

You may receive this email if you are a GCC 1.0 user or a part of the SHIP-HATS account migration, who tried to sign up for TechPass account when you already have an active account.

> If you neither re-attempted to sign up for TechPass account nor part of the SHIP-HATS account migration , this could be a potential hack and hence [report this incident][service-request] to us and monitor your TechPass account for suspicious activities.

Reviewer: I'm trying to remove the statement this is a potential hack. It's a possibility but may not be. so we just need get the user to monitor their activities without explicitly stating it is.

Author: yeah you are right. Done!
